libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.m

```diff
@@ -97,4 +141,53 @@ + (void)setSessionManager:(id<SFNetworkSessionManaging>)manager {
     kSFNetworkManager = manager;
 }
 
++ (NSDictionary *)sharedSessions {
+    return [[self mutableSharedSessions] dictionary];
+}
+
++ (void)removeSharedEphemeralSession {
+    [[self class] removeSharedSessionForIdentifier:kSFNetworkEphemeralSessionIdentifier];
```

Comment:
Can be `[SFNetwork removeSharedSessionForIdentifier:kSFNetworkEphemeralSessionIdentifier];`. Remove usage of `self` from this class method.